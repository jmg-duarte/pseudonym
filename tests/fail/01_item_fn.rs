use pseudonym::alias;

#[alias(b, c,)]
fn a() -> i32 {
    5
}

fn main() {
    assert_eq!(a(), 5);
    assert_eq!(b(), 5);
    assert_eq!(c(), 5);
}