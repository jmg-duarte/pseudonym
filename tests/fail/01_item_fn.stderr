error: unexpected end of input, expected identifier
 --> tests/fail/01_item_fn.rs:3:1
  |
3 | / #[alias(
4 | |     b,
5 | |     deprecated(c),
6 | |     deprecated(d, since = "0.1.0"),
7 | |     deprecated(e, note = "deprecation note"),
8 | |     deprecated(f, since = "0.1.0", note = "deprecation note"),
9 | | )]
  | |__^
  |
  = note: this error originates in the attribute macro `alias` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find function `a` in this scope
  --> tests/fail/01_item_fn.rs:15:16
   |
15 |     assert_eq!(a(), 5);
   |                ^ not found in this scope

error[E0425]: cannot find function `b` in this scope
  --> tests/fail/01_item_fn.rs:16:16
   |
16 |     assert_eq!(b(), 5);
   |                ^ not found in this scope

error[E0425]: cannot find function `c` in this scope
  --> tests/fail/01_item_fn.rs:17:16
   |
17 |     assert_eq!(c(), 5);
   |                ^ not found in this scope

error[E0425]: cannot find function `d` in this scope
  --> tests/fail/01_item_fn.rs:18:16
   |
18 |     assert_eq!(d(), 5);
   |                ^ not found in this scope

error[E0425]: cannot find function `e` in this scope
  --> tests/fail/01_item_fn.rs:19:16
   |
19 |     assert_eq!(e(), 5);
   |                ^ not found in this scope

error[E0425]: cannot find function `f` in this scope
  --> tests/fail/01_item_fn.rs:20:16
   |
20 |     assert_eq!(f(), 5);
   |                ^ not found in this scope
